packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -198,15 +198,33 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin {
 				}
 
 				config = workerConfig;
+
+				if (workerConfig.configPath) {
+					const dotDevDotVarsContent = getDotDevDotVarsContent(
+						workerConfig.configPath,
+						resolvedPluginConfig.cloudflareEnv
+					);
+					// if present we save a .dev.vars file onto the worker's build output
+					// directory so that it will be then detected by `vite preview`
+					if (dotDevDotVarsContent) {
+						this.emitFile({
+							type: "asset",
+							fileName: ".dev.vars",
+							source: dotDevDotVarsContent,
+						});
+					}
+				}
 			} else if (this.environment.name === "client") {
 				const assetsOnlyConfig = resolvedPluginConfig.config;
 
 				assetsOnlyConfig.assets.directory = ".";
 
+				const filesToAssetsIgnore = ["wrangler.json", ".dev.vars"];
+
 				this.emitFile({
 					type: "asset",
 					fileName: ".assetsignore",
-					source: "wrangler.json",
+					source: `${filesToAssetsIgnore.join("\n")}\n`,
```

Comment:
is there any benefit in having a variable here (i.e. do we expect to add more files)
Otherwise it looks looks inlining would be less convoluted

Reply:
the benefit of the variable are:
 - it is nicer to have if we can indeed add more files in the future
 - if is a self documenting variable, the variable name helps (at least to me) understanding that it does
 
 those are not huge benefits I don't mind inlining the array if that's what you strongly prefer